### DISCUSSION TITLE
PDF to epub conversion option

### DISCUSSION THREAD

**Discussion by github-catch-22** [Stirling-Tools/Stirling-PDF#2067] [Ideas]
Hi all
Is anyone keen to implement a pdf to epub conversion option for Stirling-PDF. There is [pdf2epubEX](https://github.com/dodeeric/pdf2epubEX) which uses [pdf2htmlEX](https://github.com/pdf2htmlEX/pdf2htmlEX) that might be a good place to start and or integrate?

What are people's thoughts on PDF -> epub? I noticed some mention of epub in the source, but there is not much beyond mentions in arrays that I could see.

I do have some experience with Java development and have had a quick look at the source code, which looks very nice, but I have no experience helping out on an open-source project.

**Comment by Frooodle** [comment]
We originally used calibre but it has issues with alpine docker images, since then we had put it on hold

**Reply by applesoff** [reply]
any updates on this?

**Reply by applesoff** [reply]
I don't see an option for this still. Is it still on hold?
